Framework/Algorithms/test/TransposeTest.h

```diff
@@ -15,6 +15,8 @@
 #include "MantidDataHandling/LoadRaw3.h"
 #include "MantidDataObjects/RebinnedOutput.h"
 
+#include <boost/math/special_functions/fpclassify.hpp>
```

Comment:
```suggestion
```
There are no other changes in this class, I believe this is not required (?).

Reply:
It is required to be able to build on mac. It is a missing header, just other compilers figure that out.